.github/workflows/codeql-analysis.yml

```
@@ -13,10 +13,10 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ master, v19, v20, v21, v22 ]
+    branches: [ master, v20, v21, v22, v23, make-v24 ]
```

- [ ] make v23 branch before merging

```suggestion
    branches: [ master, v20, v21, v22, v23 ]
```